### DISCUSSION TITLE
RetroTime N64 won't update to 1.9.2

### DISCUSSION THREAD

**Discussion by Seroczynski** [darthcloud/BlueRetro#963] [Q&A] [answered]
Hi,

I'm trying to update my RetroTime N64 BlueRetro adapter from 1.8.3 to 1.9.2. I've downloaded the zip, extracted it, select the N64 .bin, let it go to 100% and wait for it to disconnect. I've done this four times now, but the page **still says I have 1.8.3 installed**. What's going on?

Once connected this is how it looks:
![n64_blueretro_update_1](https://github.com/darthcloud/BlueRetro/assets/10141349/17ff4cec-e360-4f12-be88-404978738bc1)

And here you can see the update should have been installed successfully:
![n64_blueretro_update_2](https://github.com/darthcloud/BlueRetro/assets/10141349/6925862f-fdc6-444a-a602-24815c74c204)

**Comment by darthcloud** [comment, marked as answer]
This a known issues not yet resolved.

Try to get the adapter and pc closer to each other, or try with a different PC. It will eventually work.

**Comment by darthcloud** [comment]
also when it reach 100% and disconnect give it a minute to finish the update (wait for led to pulse again)

**Comment by Seroczynski** [comment]
As I sat closer to the N64 with my laptop, I noticed I might have downloaded the wrong firmware. I downloaded the one at the top of the list, but I only just noticed the list is in ascending order 😅
